alacritty/src/display.rs

```diff
@@ -639,14 +645,15 @@ impl Display {
         size_info: &SizeInfo,
         message_bar_lines: usize,
         search_regex: Option<String>,
+        search_label: &str,
     ) {
         let search_regex = match search_regex {
             Some(search_regex) => search_regex,
             None => return,
         };
         let glyph_cache = &mut self.glyph_cache;
 
-        let label_len = SEARCH_LABEL.len();
+        let label_len = search_label.len();
```

Comment:
The label's len is previously used for the IME already. Computing it again here is wasteful.